.github/workflows/wheel-builder.yml

```diff
@@ -139,18 +140,33 @@ jobs:
             ABI_VERSION: 'cp39'
             DOWNLOAD_URL: 'https://www.python.org/ftp/python/3.11.3/python-3.11.3-macos11.pkg'
             BIN_PATH: '/Library/Frameworks/Python.framework/Versions/3.11/bin/python3'
-    name: "Python ${{ matrix.PYTHON.VERSION }} for ABI ${{ matrix.PYTHON.ABI_VERSION }} on macOS"
+          - VERSION: '3.13t'
+            DOWNLOAD_URL: 'https://www.python.org/ftp/python/3.13.1/python-3.13.1-macos11.pkg'
+            BIN_PATH: '/Library/Frameworks/PythonT.framework/Versions/3.13/bin/python3.13t'
+    name: "Python ${{ matrix.PYTHON.VERSION }} ${{ matrix.PYTHON.ABI_VERSION }} on macOS"
     steps:
-      - run: |
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          # The tag to build or the tag received by the tag event
+          ref: ${{ github.event.inputs.version || github.ref }}
+          persist-credentials: false
+      - name: Install Python
+        if: matrix.PYTHON.VERSION != '3.13t'
+        run: |
           curl "${{ matrix.PYTHON.DOWNLOAD_URL }}" -o python.pkg
           sudo installer -pkg python.pkg -target /
+      - name: Install Python
+        if: matrix.PYTHON.VERSION == '3.13t'
+        run: |
+          curl "${{ matrix.PYTHON.DOWNLOAD_URL }}" -o python.pkg
+          sudo installer -pkg python.pkg -applyChoiceChangesXML .github/workflows/config/macos-pkg-choices-freethreaded.xml -target /
       - uses: dtolnay/rust-toolchain@a54c7afa936fefeb4456b2dd8068152669aa8203
         with:
           toolchain: stable
           # Add the arm64 target in addition to the native arch (x86_64)
           target: aarch64-apple-darwin
 
-      - run: ${{ matrix.PYTHON.BIN_PATH }} -m venv venv
+      - run: python -m venv venv
```

Comment:
Is this correct? Won't this use the system python, and not the one we just installed?

Reply:
The installer adds a symlink that ends up in the $PATH, but this is a good point and there's no reason to change this I think. I don't remember why I did it.